scripts/circle-ci/run-circle-tests.sh

```diff
@@ -7,6 +7,6 @@ SHARED=':atlasdb-tests-shared:check'
 
 case $CIRCLE_NODE_INDEX in
     0) ./gradlew --continue check -x $CASSANDRA -x $SHARED ;;
-    1) ./scripts/ete.sh ;;
+    1) ./gradlew docker $CASSANDRA --parallel --console=plain --continue ;;
```

Comment:
There is an additional problem here.  By failing to run check we bypass the additional tasks that have been associated with check (ie. licenseTest, checkstyle, and soon findbugs).  I would advocate for increasing the parallelization here to run gradlew check cassandra.